.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  pypi-publish:
+    name: upload release to PyPI
+    runs-on: ubuntu-latest
+    # Specifying a GitHub environment is optional, but strongly encouraged
+    environment: release
+    permissions:
+      # IMPORTANT: this permission is mandatory for trusted publishing
+      id-token: write
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
```

Comment:
There is no matrix build to begin with...

Reply:
But... It [works](https://github.com/pomponchik/fazy/actions/runs/7440628260/job/20241836529)!

Reply:
Missing substitution value is replaced with empty string. According to https://github.com/actions/setup-python documentation:
> The default version of Python or PyPy in PATH varies between runners and can be changed unexpectedly so we recommend always setting Python version explicitly